lib/process/local-repl-process.coffee

```diff
@@ -132,7 +131,6 @@ class LocalReplProcess
 
   interrupt: ->
     @conn.interrupt()
-    @appendText("Interrupting")
```

Comment:
I had added a bunch of @appendText messages and when I was cleaning them I removed this by mistake.